src/services/embargo.ts

```diff
@@ -80,26 +80,25 @@ const getEmbargoStageForPublishDate = (firstPublishedAt?: Date): EmbargoStage =>
  * @param dbFile
  */
 export const getEmbargoStage = (dbFile: File): EmbargoStage => {
-  logger.debug(`${dbFile.fileId}: Recalculating embargo stage`);
-
-  // a releaseState of PUBLIC means this is already public, return EmbargoStage of PUBLIC
-  if (dbFile.releaseState === FileReleaseState.PUBLIC) {
-    logger.debug(`${dbFile.fileId}: File is already public.`);
-    return EmbargoStage.PUBLIC;
-  }
```

Comment:
This safeguard is no longer required. We always want this method to calculate the expected embargo stage, it is the invoking method that determines what the state for the file should be.